example/index.tsx

```diff
@@ -25,11 +25,25 @@ import {
 import { getTheme } from '../src/ui/theme';
 import usePublicationSW from '../src/ServiceWorker/client';
 import AxisNowEncrypted from './axisnow-encrypted';
+import htmlStyles from 'url:../src/HtmlReader/injectable-styles.css';
```

Comment:
This is one way to import a css file as a url which works in parcel. In webpack you would use `file-loader`